recipes/gw/fix-ldflags.patch

```diff
@@ -0,0 +1,5 @@
++++ b/Makefile
+@@ -55,1 +55,1 @@
+-        LDFLAGS=
++	       
```

Comment:
What is the idea here ?
At https://github.com/bioconda/bioconda-recipes/pull/49007/files#diff-6083b4eea3f9c531df6b988ac53e627ce15e9a85afb1da658d70811b908384bfR8 you pass `LDFLAGS` to `make`, i.e. you overwrite the default (empty) value anyway.
IMO there is no need of this patch.

Reply:
This is for a different PR on the termux packages repo. Setting LDFLAGS clears out anything passed in to make